releases/release-1.13/release_team.md

```diff
@@ -1,12 +1,12 @@
 | **Role** | **Name** (**GitHub / Slack ID**)  | **Shadow Name(s) (GitHub / Slack ID)** |
 | ------ | ------ | ------ |
 | Lead | Aishwarya Sundar ([@AishSundar](https://github.com/AishSundar)) | Aaron Crickenberger ([@spiffxp](https://github.com/spiffxp)) |
-| Enhancements | Kendrick Coleman ([@kacole2](https://github.com/kacole2)) | Michael Singh ([@wadadli](https://github.com/wadadli)), Guinevere Saenger ([@guineveresaenger](https://github.com/guineveresaenger) / @gsaenger on Slack), Arnaud Meukam ([@ameukam](https://github.com/ameukam)) |
+| Enhancements | Kendrick Coleman ([@kacole2](https://github.com/kacole2)) | Michael Singh ([@wadadli](https://github.com/wadadli)), Guinevere Saenger ([@guineveresaenger](https://github.com/guineveresaenger) / @gsaenger on Slack), Arnaud Meukam ([@ameukam](https://github.com/ameukam)), Claire Laurence ([@claurence](https://github.com/claurence)) |
 | CI Signal | Josh Berkus ([@jberkus](https://github.com/jberkus)) | Shivaani Mall ([@shivani1494](https://github.com/shivani1494)), Morten Torkildsen ([@mortent](https://github.com/mortent)) |
```

Comment:
From the GH issue, can we please add TaoBeier (@tao12345666333) and Maria Ntalla (@mariantalla) as shadows for CI Signal?